tests/integration/java_integration_test.go

```diff
@@ -19,10 +20,11 @@ func TestIntegrations(t *testing.T) {
 	t.Run("stack-reference", func(t *testing.T) {
 		dir := filepath.Join(getCwd(t), "stack-reference")
 		test := getJavaBase(t, integration.ProgramTestOptions{
-			Dir:           dir,
-			Quick:         true,
-			DebugUpdates:  false,
-			DebugLogLevel: 0,
+			Dir:              dir,
+			Quick:            true,
+			DestroyOnCleanup: true,
```

Comment:
I was convinced by the part about `more realistic stack reference testing` that this might be helpful but maybe I just don't understand this feature. If it is not helpful then I'll ofc remove it.